### DISCUSSION TITLE
please assist trying to get 200 okay msg register

### DISCUSSION THREAD

**Discussion by jmcnamara10001** [baresip/baresip#3021] [Q&A]
Hi -
I am trying to get a 200 okay register msg. I notice that the ip:port -> destip:port the first port is not 5060 but the destip port is 5060.
it indicates:
register sip:voip.ms 
via sip 2
contact
to acctnum@voip.ms
from acctnum@voip.ms
call-id
cseq register
user-agent baresip
allow: invite ack, bye etc

i dont know why it wont register! please help
thanks

**Comment by cspiel1** [comment]
Please start baresip with `-s` option and paste the output in a code block here!

**Comment by jmcnamara10001** [comment]
Hi I got it to register and can make an echo call but it is very noisy.
I am gong to use a codec capable of variable bit rates like opus to match the bitrate of the call or get it closer than g.729.
I found out I cant turn down the sound of the modem on my phone very easy which is what i use to test it but after it connects about 20 seconds in it starts feedback and screeching so i will try to clean it up a bit. Thanks for making baresip.
I will keep playing on AI until I get it hopefully. Down the road i might pick up a real handset and a base with pass through vpn capable.
thanks,
jim
